### PR TITLE
TELCODOCS-1017: Image registry and image source policy setup in disco…

### PR DESCRIPTION
**Pull request by slovern**
…nnected environment for TALM precaching/backup features

[TELCODOCS-1017](https://issues.redhat.com//browse/TELCODOCS-1017): Image registry and image source policy setup in disconnected environment for TALM precaching/backup features

Version(s):

Version(s): 4.12, 4.11, 4.10

Issue: https://issues.redhat.com/browse/TELCODOCS-1017

Link to docs preview: https://53711--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ztp-acm-installing-disconnected-rhacm.adoc [hunk 1]
```diff
@@ -15,6 +15,11 @@ Use {rh-rhacm-first}, {gitops-title}, and {cgu-operator-first} on the hub cluste
 * You have logged in as a user with `cluster-admin` privileges.
 
 * You have configured a disconnected mirror registry for use in the cluster.
++
+[NOTE]
+====
+The disconnected mirror registry that you create must contain an image version that matches the version of {cgu-operator-first} running in the hub cluster. The disconnected mirror registry that you specify must be accessible from the spoke cluster.
```
Comment (imiller0) [inline modules/ztp-acm-installing-disconnected-rhacm.adoc:21]: I think this needs a bit more context. The images specified here are the precaching and backup job images. Something like this perhaps?
```suggestion
The disconnected mirror registry that you create must contain a version of TALM backup and precache images that match the version of {cgu-operator-first} running in the hub cluster. The spoke cluster must be able to resolve these images in the disconnected mirror registry.
```

Comment (slovern) [reply modules/ztp-acm-installing-disconnected-rhacm.adoc]: ok. have changed accordingly

**Review by imiller0 — COMMENTED** [review]
lgtm

**Comment by josclark42** [comment]
lgtm

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**